--- lib/main_online.dart
+++ lib/main_online.dart
@@ -2952,7 +2952,7 @@
                     const SizedBox(height: 10),
                     const Align(
                       alignment: Alignment.centerRight,
-                      child: Text('v71',
+                      child: Text('v72',
                           style: TextStyle(
                               fontSize: 11,
                               color: Color(0xFF8A98A8),
